.azure-pipelines/build-plcrashreporter-1ES.yml

```diff
@@ -48,7 +48,7 @@ extends:
             sbomEnabled: false
         steps:
         - checkout: self
-        - bash: 'brew install doxygen graphviz protobuf-c'
+        - bash: 'brew install doxygen graphviz'
```

Comment:
> we already have this package in the repository

No, the repo doesn't contain it. It was used to regenerate sources from `*.proto` files via `protoc-c` tool. See this build rule - https://github.com/microsoft/plcrashreporter/blob/1aed8f7dc79ce8e674c61e430ef51ca3db18cea9/CrashReporter.xcodeproj/project.pbxproj#L778

Reply:
We use this [package](https://github.com/microsoft/plcrashreporter/tree/master/Dependencies/protobuf-c/protobuf-c) which is stored in the repository

Reply:
I know. It's the library sources, but not `protoc-c` binary for building

Reply:
Thanks for the clarification, I will edit the PR description, but this package installation is still not required for the PLCrashReporter build.

Reply:
Yes, it's optional for the build in general - if there is no `protoc-c` on build machine, the build of `*.proto` files is skipped.
But it's important to have in on CI to make sure that there are no unapplied changes in `*.proto` files were made.